NuGet.config

```diff
@@ -16,7 +16,9 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsPackages" value="..\bin\Debug\NuGet" />
```

Comment:
This is actually required to get the AspxCore test application to reference the locally built DependencyCollector NuGet package. I think this won't be necessary after you guys move to VS 2017, but I'm not positive.